docs/mixins.md

```diff
@@ -43,6 +43,12 @@ ThreadSafeCreateMixin
 The inherited class will return a new instance of itself in every call of
 `create`.
 
+ThreadSafeWithArgsCreateMixin
+-----------------------------
+
+The inherited class will return a new instance of itself in every call of
+`create` but with this mixin you can pass args and kwargs do initialize the instance.
+
```

Comment:
What about adding
```suggestion
**caveat**: When `ThreadSafeWithArgsCreateMixin` is used, every backend of the same type should use it as well, so that the backend pool can create every backend instance using the exact same parameters.

```
?

An example of usage of a `BackendPool` which have backends using `ThreadSafeWithArgsCreateMixin` might be good as well.